src/track/keyutils.cpp

```diff
@@ -247,8 +247,17 @@ QString KeyUtils::getGlobalKeyText(const Keys& keys, KeyNotation notation) {
 }
 
 // static
-ChromaticKey KeyUtils::guessKeyFromText(const QString& text) {
-    QString trimmed = text.trimmed();
+ChromaticKey KeyUtils::guessKeyFromText(QString text) {
```

Comment:
Why the change from const-ref to by value?

Reply:
Sure, if you think reassigning in the body is necessary, its no longer const. It could probably made const again though once that code has been replaced.